src/agent/service/nodesChainDataGet.ts

```diff
@@ -1,24 +1,22 @@
 import type * as grpc from '@grpc/grpc-js';
-import type { ClaimIdEncoded } from '../../claims/types';
-import type { NodeManager } from '../../nodes';
+import type { Sigchain } from '../../sigchain';
 import type * as utilsPB from '../../proto/js/polykey/v1/utils/utils_pb';
 import { utils as grpcUtils } from '../../grpc';
 import * as nodesPB from '../../proto/js/polykey/v1/nodes/nodes_pb';
 
 /**
  * Retrieves the ChainDataEncoded of this node.
  */
-function nodesChainDataGet({ nodeManager }: { nodeManager: NodeManager }) {
+function nodesChainDataGet({ sigchain }: { sigchain: Sigchain }) {
   return async (
     call: grpc.ServerUnaryCall<utilsPB.EmptyMessage, nodesPB.ChainData>,
     callback: grpc.sendUnaryData<nodesPB.ChainData>,
   ): Promise<void> => {
     try {
       const response = new nodesPB.ChainData();
-      const chainData = await nodeManager.getChainData();
+      const chainData = await sigchain.getChainData();
       // Iterate through each claim in the chain, and serialize for transport
-      for (const c in chainData) {
-        const claimId = c as ClaimIdEncoded;
+      for (const claimId in chainData) {
```

Comment:
This is `claimIdEncoded`. @tegefaulkes please change this to:

```ts
let claimIdEncoded: ClaimIdEncoded;
for (claimIdEncoded in chainData) { ... }
```

Reply:
Fixed it.